### ISSUE TITLE
Feature request: Presets

### ISSUE THREAD

**Issue by BlackCapCoder** [mps-youtube/mps-youtube#757]
I use mps-youtube for two things- listening to music and narrations. When I listen to music I like using mplayer without any playerargs, but when I listen to narrations I like using vlc with playerargs --rate=2.1.

Currently I have to switch these two settings manually depending on what I want to listen to, but it would be much more convenient if I could toggle a preset. 

I propose having two commands, `preset load` and `preset store`, both of which would take a name as an argument. `preset store` would save the current settings to a named preset, that could be loaded with the `preset load` command.

**Comment by ZaxonXP** [comment]
Indeed, I would also use this feature.

**Comment by Vrihub** [comment]
As an alternate solution, you can pass the set parameters on the command line `mpsyt set x y, set w z,...` and use a wrapper script, or alias to automate this.
BUT: there's a bug that currently interferes with this approach: parameters set on the command line are remembered in later invocations, and they shouldn't: see #455

**Comment by ZaxonXP** [comment]
@Vrihub: I could work with this feature. I am not against the scripts, but still presets would be nice (or loading configuration from within the mpsyt) when the program is already in use.

**Comment by BlackCapCoder** [comment]
@Vrihub @ZaxonXP Indeed. Presets would be nice, but I could absolutely settle for the argument solution as in my use case I am going trough a context switch anyways.
